### PR TITLE
[MM-42769] - Fix Actions Menu tooltip `X` click and remove `Opt out` link

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR 
1. fixes the tooltip so that clicking the `x` closes the tooltip.  Previously, it did nothing and the tip remained open.
2. remvose the Opt out message discussed [here](https://community-daily.mattermost.com/core/pl/ibbgjpabj7daigfjbyh83tmjnh) asking how it should be handled.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-42769

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |

#### Release Note

```release-note

```

### CODE REVIEW

**Comment by matthewbirtch** [comment]
@jfrerich is this ready for review or is it still a draft?

**Comment by jfrerich** [comment]
It is ready for review.  I just converted it from draft.  Thanks!

**Comment by jfrerich** [comment]
> @jfrerich I think you missed updating the PR title :)

You guys and the symantics! Always joking :)

**Review by matthewbirtch — APPROVED** [review]
Seems good to me!

**Comment by DHaussermann** [comment]
@jfrerich I can still see a transparency issues when 1 of the menu items is being moused over.

1. In RHS the `help` link os showing through 
image


2. When the back color of a post is changed because of mouse-over - I can see the color bleeding through around the devider.
image

Please let me know if you can reproduce this.

**Comment by jfrerich** [comment]
@DHaussermann you'll see those issues addressed in this PR.. https://github.com/mattermost/mattermost-webapp/pull/10111

**Comment by DHaussermann** [comment]
@jfrerich sorry. I tested this is on the wrong server - Please disregard.

**Comment by michelengelen** [comment]
@jfrerich i am on PTO ... so if you want to get that in quickly feel free to remove me and add a substitute as reviewer! 👍🏻

File: components/actions_menu/actions_menu_tutorial_tip.tsx [hunk 1]
```diff
@@ -16,30 +16,35 @@ type Props = {
...
-            defaultMessage='Message actions that are provided\nthrough apps, integrations or plugins\nhave moved to this menu item.'
-        />
-    );
-    const nextBtn = (
-        <FormattedMessage
-            id={'tutorial_tip.got_it'}
-            defaultMessage={'Got it'}
-        />
-    );
+    const onDismiss = (e: React.MouseEvent) => {
```
Comment (larkox) [inline components/actions_menu/actions_menu_tutorial_tip.tsx:39]: useCallback

**Review by larkox — APPROVED** [review]
Code LGTM, just one small comment.

**Review by jwilander — APPROVED** [review]

**Comment by DHaussermann** [comment]
@jfrerich The non functional link has been removed and the `x` option works as expected to close without removing the dot!


However, I do see an issue that may be related to the PR. When I click the `Got it` button - The post that had the focus at the time remains in a "mouse over" state.

The back color is still the alternate color and the options remain visible even when I mouse away. There are several ways to clear it but it has a confusing feel. I can even use my mouse to put the focus in the input box and type.
image

Simple to setup by inviting a new user. I pulled `master` up to see if the issue may already be resolved from different Actions PR but I still see it.
I don't recall this happening before when I click `Got It` Are you able to repro this?

**Comment by jfrerich** [comment]
@DHaussermann, please give this another shot.  The issue you were seeing should be resolved.  I was also able to repro and thanks for finding this issue.

**Review by DHaussermann — APPROVED** [review]
Thanks Jason.
The issue found has been resolved. The changes mentioned above have been  successfully implemented. 

LGTM!

**Pull request merged**